en-US/snippets/2017/sep2017_a.lang

```diff
@@ -0,0 +1,124 @@
+## NOTE: Reference bug 1392702. Deadline: 4 Sept, 2017.
+## NOTE: Guidelines on localizing snippets: https://goo.gl/nEhdt4. The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: No character length limit, but try not to exceed English length.
+
+
+# Snippet for Firefox Screenshots
+# Screenshots remains in English
```

Comment:
Better: "Screenshots (uppercase) is the name of the feature and has to be kept in English" (same later)